### PR TITLE
Add Test branch

### PR DESCRIPTION
**Pull request by taiwo2**
On this branch, we (@taiwo2 and @murodjon000) tested our ruby code by adding specs to all the public methods created on the tic-tac-toe game and tested for both negative and positive.

### CODE REVIEW

File: spec/enum_spec.rb [hunk 1]
```diff
@@ -0,0 +1,202 @@
+# rubocop:disable all
+
```
Comment (peterrobert) [inline spec/enum_spec.rb]: You should not disable rubocop. Instead, you should fix the errors. I have removed that disable and a lot of errors popped up. ⬇️  Kindly fix these errors

Screenshot from 2021-01-08 12-02-22

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED 🛑 ♻️ 

Hello, @taiwo2 and @Murodjon000 👋🏿  your project looks amazing 👏🏿 you have done such a good job 👏🏿 . However, there are still some remaining issues that need to be resolved. 

- [ ] In your readme file kindly add how to run tests in this project.
- [ ] Kindly don't disable rubocop. Instead, fix all the offenses.

### if you have any questions kindly reach me on slack @peterrobert

### HAPPY CODING 💻

File: spec/enum_spec.rb (hunk 1 above)
Comment (taiwo2) [reply spec/enum_spec.rb]: will be fixed as expected

**Comment by mimipeshy** [comment]
## STATUS :APPROVED :partying_face:  :fireworks: 

Hey team @taiwo2 and @Murodjon000 :wave: :wave: good job on working on the feedback given. The test cases pass well. This project is therefore approved. Go ahead and merge, all the best in the next projects :smile: 

Happy Coding :computer: 

Regards
Peris Ndanu

**Pull request merged**